synapse/replication/http/_base.py

```diff
@@ -94,6 +95,8 @@ class ReplicationEndpoint(metaclass=abc.ABCMeta):
     METHOD = "POST"
     CACHE = True
     RETRY_ON_TIMEOUT = True
+    RETRY_ON_CONNECT_ERROR = True
+    RETRY_ON_CONNECT_ERROR_ATTEMPTS = 5  # =63s (2^6-1)
```

Comment:
please could you add a couple of words to the docstring above, where these attributes are documented?

Reply:
https://github.com/matrix-org/synapse/pull/12182/commits/bde45663d04bc04902654abe457faaabb97ac136